webapp/channels/src/components/user_list_row/user_list_row.tsx

```diff
@@ -79,11 +79,12 @@ const UserListRow = ({user, status, extraInfo = [], actions = [], actionProps, a
     let statusProp: string | undefined;
     if (extraInfo && extraInfo.length > 0) {
         emailProp = (
-            <FormattedMarkdownMessage
-                id='admin.user_item.emailTitle'
-                defaultMessage='**Email:** {email}'
+            <FormattedMessage
+                id='admin.user_item.emailTitle_admin'
```

Comment:
Why the admin suffix?

Reply:
I did that based on the instruction on the issue:

```...change the message ID after the update as the current extraction tool does not automatically detect changes in the messages.```

Reply:
Your ID suggestion looks fine. I'll make the changes. Thank you!

Reply:
I have made the changes, @larkox.